src/Symfony/Component/Validator/Constraints/PasswordStrength.php

```diff
@@ -40,6 +41,13 @@ final class PasswordStrength extends Constraint
 
     public int $minScore;
 
+    /**
+     * @param array<string,mixed>|null $options  The options as an associative array
+     * @param self::STRENGTH_*|null    $minScore The minimum required strength of the password
```

Comment:
```suggestion
     * @param self::STRENGTH_*|null    $minScore The minimum required strength of the password (STRENGTH_MEDIUM by default)
```

Reply:
Shoud I document all the defaults in all constraints too?

Reply:
Ok I'll do another pass!